frontend/include/chpl/uast/FunctionSignature.h

```diff
@@ -17,8 +17,8 @@
  * limitations under the License.
  */
 
-#ifndef CHPL_UAST_FUNCTION_SIGNATURE_H
-#define CHPL_UAST_FUNCTION_SIGNATURE_H
+#ifndef CHPL_UAST_FUNCTIONSIGNATURE_H
+#define CHPL_UAST_FUNCTIONSIGNATURE_H
```

Comment:
FWIW, I think the name of this class is confusing, because it sounds like a Type. Maybe it should be "AnonymousFunction" or something like that?